scripts/readme-gen/readme_gen.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
```

Comment:
Do we want to have copies of readme-gen everywhere? How do we maintain changes?

Reply:
My pending synthtool PR adds readmegen into the synthtool. Eventually we can remove this separate script and only do sample README regens with the templates in synthtool.